src/main/java/org/testng/internal/Parameters.java

```diff
@@ -200,6 +257,21 @@ private static void checkParameterTypes(String methodName,
     }
   }
 
+  private static String prettyFormat(List<Class> classes) {
+    StringBuilder builder = new StringBuilder("<");
+    if (classes.size() == 1) {
```

Comment:
Yes, but in case of `size == 1`, the loop has no element and only  `builder.append(classes.get(length-1).getSimpleName());` will be triggered, right?

At least, add `.getSimpleName()` in case of 1 and maybe return an empty string in case of 0.